tests/haskell-lexeme-tests.el

```diff
@@ -292,6 +292,26 @@ buffer."
    "[xml| <xml />"
    '("[xml| <xml />")))
 
+(ert-deftest haskell-lexeme-quasi-quote-qual-1 ()
```

Comment:
These tests pass even without my fix for some reason, not 100% sure if it makes sense to have them here